apm-agent-plugins/apm-aws-sdk/apm-aws-sdk-common/src/test/java/co/elastic/apm/agent/awssdk/common/AbstractAwsClientIT.java

```diff
@@ -55,7 +55,7 @@ public abstract class AbstractAwsClientIT extends AbstractInstrumentationTest {
 
     public AbstractAwsClientIT(String localstackVersion) {
         DockerImageName localstackImage = DockerImageName.parse("localstack/localstack:" + localstackVersion);
-        localstack = new LocalStackContainer(localstackImage).withServices(localstackService());
+        localstack = new LocalStackContainer(localstackImage).withServices(localstackService()).withEnv("SQS_ENDPOINT_STRATEGY", "dynamic");
```

Comment:
seems the most flexible per https://docs.localstack.cloud/user-guide/aws/sqs/#queue-urls